libraries/ESP32/examples/Camera/CameraWebServer/app_httpd.cpp

```diff
@@ -1304,3 +1315,14 @@ void startCameraServer()
         httpd_register_uri_handler(stream_httpd, &stream_uri);
     }
 }
+
+void ledc_setup()
+{
+#ifdef CONFIG_LED_ILLUMINATOR_ENABLED
+  // PWM
+  ledcSetup(LEDC_CHANNEL_0, 4000, LEDC_TIMER_8_BIT);
+  ledcAttachPin(FLASH_LED_GPIO, LEDC_CHANNEL_0);
+#else
+  ESP_LOGI(TAG, "Flash led illumination is not configured");
```

Comment:
Please use Arduino function `log_i(...)` instead of IDF `ESP_LOGI()`

Reply:
> Please use Arduino function log_i(...) instead of IDF ESP_LOGI()

There are more then one line with ESP_LOGI (I think example was ported from IDF), so all must be changed or only this one or maybe leave as is? :)

Reply:
I see... it is used all over in app_httpd.cpp.
If possible, it should be changed everywhere.

Reply:
Let's keep it simple for now. Keep it using `ESP_LOGI()`